airflow/providers/amazon/aws/operators/eks.py

```diff
@@ -26,7 +26,13 @@
 from airflow import AirflowException
 from airflow.models import BaseOperator
 from airflow.providers.amazon.aws.hooks.eks import EksHook
-from airflow.providers.cncf.kubernetes.operators.kubernetes_pod import KubernetesPodOperator
+
+try:
+    from airflow.providers.cncf.kubernetes.operators.kubernetes_pod import KubernetesPodOperator
+except ImportError as e:
+    from airflow.exceptions import AirflowOptionalProviderFeatureException
+
+    raise AirflowOptionalProviderFeatureException(e)
```

Comment:
Hmmmm... It appears that if user did not install `cncf.kubernetes` extra then all other operators in this module unavailable.

All other operators, as far as I know, use AWS API. So is it better move this operator into separate module as follow up?

Reply:
You cant really use eks module without cncf so probably most users has cncf already installed

Reply:
I'm not sure is it available into provider but also EKS could use for run EMR jobs as well as compute environment for run AWS Batch Job (we definitely can't do it right now).

Reply:
It is just thinking out loud, I really do not know is eks operators provide something more then use it with `EksPodOperator`